plugins/connectwise/icon_connectwise/util/endpoints.py

```diff
@@ -0,0 +1,6 @@
+BASE_URL = "https://api-na.myconnectwise.net/v4_6_release/apis/3.0"
```

Comment:
Connectwise can be onprem. So shouldnt this come from connection parameters input